flytekit/clis/sdk_in_container/basic_auth.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import absolute_import
+
+import base64 as _base64
+import logging as _logging
+
+import requests as _requests
+
+from flytekit.common.exceptions.base import FlyteException as _FlyteException
+from flytekit.configuration.creds import (
+    CLIENT_CREDENTIALS_SECRET_LOCATION as _CREDENTIALS_SECRET_FILE,
+    CLIENT_CREDENTIALS_SECRET as _CREDENTIALS_SECRET,
+)
+
+_utf_8 = 'utf-8'
+
+
+class FlyteAuthenticationException(_FlyteException):
```

Comment:
NIt: should we define this in flyte.exceptions.user?

Reply:
@wild-endeavor can we use https://github.com/lyft/flytekit/pull/23/files#diff-7fbb98dd27586954c518f84994ed987cR77?